common/src/main/scala/explore/components/Tile.scala

```diff
@@ -49,8 +52,6 @@ object Tile {
       .stateless
       .render_PC { (p, c) =>
         ResizeDetector() { s =>
```

Comment:
Should we remove this line, since `ResizeDetector` is now invoked within `ResponsiveComponent` below?

Reply:
Indeed I removed it but I think it came back on the rebase